YoutubeExplode/YoutubeClient.cs

```diff
@@ -40,8 +40,17 @@ public class YoutubeClient
     /// </summary>
     public YoutubeClient(HttpClient http, IReadOnlyList<Cookie> initialCookies)
     {
-        var youtubeHttp = new HttpClient(new YoutubeHttpHandler(http, initialCookies), true);
+        // Add necessary cookies.
+        var cookies = new List<Cookie>(initialCookies)
+        {
+            new("SOCS", "CAESEwgDEgk0ODE3Nzk3MjQaAmVuIAEaBgiA_LyaBg") { Domain = "youtube.com" } //cookie consent request bypass
```

Comment:
Is this value static?

Reply:
Yes it is. I've decided to take the same value as reported on the stackoverflow post (it works after ~1 year), instead of use another generated by a my session. To have a dynamic value, generated by js interaction, is out of my current competence.